capstone/src/main/webapp/dashboard-loader.js

```diff
@@ -28,15 +28,20 @@ function createRoomDivElement(room) {
             <div class="card margin margin-left">
                 <img class="card-img-top" src="/assets/soundwave.svg" alt="Room Card">
                 <div class="card-body text-center">
-                    <h5 class="card-title">${room.entity.propertyMap.title}</h5>
+                    <h5 class="card-title" id="room-title"></h5>
                     <a class="card-text small-text" href="#">Kyle Smith</a>
                     <div class="card-text small-text">Followers: Infinite</div>
                     <div class="card-text small-text">Tag(s): </div>
                     <div class="small-spacing-bottom"></div>
-                    <button type="button" class="btn btn-default" 
-                    onclick="window.location.href='lesson.html?room_id='+${room.entity.key.id}">View</button>
+                    <div class="btn btn-default" id="room-link"></button>
```

Comment:
<div ...></button>? This doesnt freak the browser out? Most browsers are pretty forgiving, but Im surprised this works. In any case, this should be an <a> anyway (per below, changed in a separate PR)

Reply:
What should this have been? Like why did you expect it to freak the browser out?

Reply:
<div></button>

Reply:
I didn't notice and the browser didn't catch it I guess. I'll change it now so I can merge